src/main.js

```diff
@@ -1,8 +1,10 @@
 import './scss/style.scss';
 import Wrapper from './layouts/MainWrapper';
+import Modal from './components/Modal';
```

Comment:
Czy tego modala nie powinniśmy importować w MainSection.js?

Reply:
Dobra, przemyślałem to, lepiej jak jest tutaj, bo modal przykrywa nam wszystko (MainSection, LogoSection, etc.).